common/models/signer.json

```diff
@@ -23,7 +23,11 @@
       "type": "date"
     }
   },
-  "hidden": ["email"],
+  "scope": {
+    "order": "name ASC",
+    "limit": 100
+  },
+  "hidden": ["id", "email"],
```

Comment:
@sgimeno We should hide `created_at` as well.

Reply:
Its hidden by default